roles/ceph-common/tasks/checks/check_mandatory_vars.yml

```diff
@@ -22,6 +22,21 @@
   tags:
     - package-install
 
+- name: make sure ceph_stable_release is set
+  fail:
+    msg: "make sure ceph_stable_release is set to a release name, http://docs.ceph.com/docs/jewel/releases/"
```

Comment:
Maybe we should just list which releases we support here instead of linking to the docs? Which I think is jewel, kraken and now luminous.

Those docs don't mention kraken or luminous anyway.